loopslab.js

```diff
@@ -0,0 +1,64 @@
+// Number 1a Write a while loop and a for loop that takes the variable "num" and logs all the numbers,
+//descending, between "num" and 1.
+//let num = 15;
+//while (num >= 1) {
+ //console.log (num);
+ //num --
+//}
+
+
+// //1b
+//for (let num = 15; num >= 1; num --) {
+//console.log (num);
+//}
+
+//
+// Number 2 Write a while loop and a for loop that takes the variable "max",
+//and iterates over all numbers from 0 to "max". For each iteration,
+//it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+//
+//
+//let max = 20;
```

Comment:
This is console logging numbers in descending order 20 to 0. The question asks for increasing order displaying the numbers 0 to 20.

Reply:
Please reverse the order